apps/dashboard/app/models/launcher.rb

```diff
@@ -335,9 +335,12 @@ def add_default_fields(form: [], **_args)
   def add_script_to_form(form: [], attributes: {})
     form << 'auto_scripts' unless form.include?('auto_scripts')
 
-    attributes[:auto_scripts] = {
-      directory: project_dir
-    }
+    dir = { directory: project_dir }
+    attributes[:auto_scripts] = if attributes[:auto_scripts]
+      attributes[:auto_scripts].merge(dir)
+    else
+      dir
+    end
```

Comment:
I don't know what IDE you're using, but if you use vscode at OSC you should use the solargraph plugin to lint.

You'll see stuff like this 
![image](https://github.com/user-attachments/assets/5b3cd351-74fe-4b17-860b-8c5b9a170183)

where it's telling you to reformat this section to this
![image](https://github.com/user-attachments/assets/566594dc-0836-4426-a6c3-f39674af82e1)

Reply:
Thanks for pointing that out, I did have it installed but something was up with the configuration.